tests/nfv/prepare_env.pm

```diff
@@ -43,48 +43,50 @@ sub run {
     my ($self)         = @_;
     my $vsperf_repo    = "https://gerrit.opnfv.org/gerrit/vswitchperf";
     my $vsperf_version = get_required_var('VSPERF_VERSION');
-    my $dpdk_repo      = "http://dpdk.org/git/dpdk";
     my $vnf_image      = get_required_var('VNF_IMAGE');
     my $trafficgen_ip  = get_trafficgen_ip();
     my $children       = get_children();
     my $child_id       = (keys %$children)[0];
 
     record_info("INFO", "Install needed packages for NFV tests: OVS, DPKD, QEMU");
-    zypper_call('--quiet in git-core openvswitch-switch dpdk qemu tcpdump', timeout => 200);
+    zypper_call('--quiet in git-core openvswitch-switch dpdk qemu tcpdump', timeout => 60 * 4);
```

Comment:
I do get why you change timeouts to 60 * 4, but is there a reason to change the effective timeouts throughout the code?

Reply:
some of them have been modified a bit, yes. The reason is timeouts in OSD again.. but in this concrete case I didn't want to write 60 * 3.33333  :)